lesson1/src/stacks_queues/stack.js

```diff
@@ -38,6 +52,7 @@ class Stack {
    */
   isEmpty() {
     // todo: do stuff
+    return this.top<=0?true:false
```

Comment:
this seems a little redundant... `this.top<=0` is already a boolean  so `return this.top<=0` should work the same way